src/gridstack.jQueryUI.js

```diff
@@ -31,7 +31,7 @@
   JQueryUIGridStackDragDropPlugin.prototype.resizable = function(el, opts) {
     el = $(el);
     if (opts === 'disable' || opts === 'enable' || opts === 'destroy') {
-      el.resizable(opts);
+      el.resizable().resizable(opts);
```

Comment:
thank you for the fix! but it has the side effect of enabling resizable before destroying it, which is wasteful if it wasn't created to start with. looking to see if we can avoid this...